app/renderer/reducers/frameReducer.js

```diff
@@ -112,26 +115,28 @@ const frameReducer = (state, action, immutableAction) => {
       }
 
       // TODO fix race condition in Muon more info in #9000
```

Comment:
this TODO can be removed now. Can you please remove it in the follow up and you can close #9000 with it? :smiley: